collections/_article/dev-snapshot-godot-4-5-dev-5.md

```diff
@@ -0,0 +1,140 @@
+
+### Web: WebAssembly SIMD support
+
+As you might recall, Godot 4.0 initially released under the assumption that multi-threaded web support would become the standard, and only supported that format for web builds. This assumption unfortunately proved to be wishful thinking, and was [reverted in 4.3](/article/dev-snapshot-godot-4-3-dev-3/#single-threaded-web-exports) by allowing for single-threaded builds once more. However, this doesn't mean that these single-threaded environments are inherently incapable of parallel processing; it just requires alternative implementations. One such implementation, <abbr title="Single instruction, multiple data">[SIMD](https://en.wikipedia.org/wiki/Single_instruction,_multiple_data)</abbr>, is a perfect candidate thanks to its support across [all major browsers](https://caniuse.com/wasm-simd). To that end, web-wiz [Adam Scott](https://github.com/adamscott) has taken to integrating this implementation for our web builds by default ([GH-106319](https://github.com/godotengine/godot/pull/106319)).
+
+### Inline color pickers
+
+While it's always been possible to see what kind of variable is assigned to an exported color in the inspector, some users have expressed a keen interest in allowing for this functionality within the script editor itself. This is because it would mean seeing what kind of color is represented by a variable without it needing to be exposed, as well as making it more intuitive at a glance as to what color a name or code corresponds to. [Koliur Rahman](https://github.com/dugramen) has blessed us with this quality-of-life goodness, which adds an inline color picker [GH-105724](https://github.com/godotengine/godot/pull/105724). Now no matter where the color is declared, users will be able to immediately and intuitively know what is actually represented in a non-intrusive manner.
+
+<img src="/storage/blog/dev-snapshot-godot-4-5-dev-5/inline-color-picker.webp" alt="Inline color picker"/>
+
+### Rendering goodies
+
+The renderer got a fair amount of love this snapshot; not from any one PR, but rather a multitude of community members bringing some long-awaited features to light. [Raymond DiDonato](https://github.com/RGDTAB) helped SMAA 1x make its transition from addon to fully-fledged engine feature ([GH-102330](https://github.com/godotengine/godot/pull/102330)). [Capry](https://github.com/LunaCapra) brings bent normal maps to further enhance specular occlusion and indirect lighting ([GH-89988](https://github.com/godotengine/godot/pull/89988)). Our very own [Clay John](https://github.com/clayjohn) converted our Compatibility backend to use a fragment shader copy instead of a blit copy, working around common sample rate issues on mobile devices ([GH-106267](https://github.com/godotengine/godot/pull/106267)). More technical information on these rendering changes can be found in their associated PRs.
```

Comment:
These (SMAA and bent normals) should be accompanied by before and after images. There are some very nice examples in the relevant PRs. 

Especially bent normals which is very impressive